discord/message.go

```diff
@@ -365,6 +368,16 @@ func (m MessageApplication) CreatedAt() time.Time {
 	return m.ID.Time()
 }
 
+const (
+	// A standard reference used by replies.
+	MessageReferenceTypeDefault = iota
```

Comment:
Should this be named `MessageReferenceTypeReply` instead?

Reply:
@diamondburned Discord API defines type `1` as the default message reference type: https://discord.com/developers/docs/resources/message#message-reference-types

Reply:
We've had other constants in Arikawa named differently from the documentation just because it's more obvious that way. I think this is one of those cases.

Reply:
This constant is also untyped. This should be fixed.

Reply:
This isn't resolved yet, but I'm choosing to keep the name as `Default`. I'll follow this up with a separate commit.